### PR TITLE
DM-46084: Apply responsive class to updated images

### PR DESCRIPTION
**Pull request by ugyballoons**

### CODE REVIEW

File: src/js/pages/current.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ import { WebsocketClient } from '../modules/ws-service-client'
     _getById('seqNum').textContent = newEvent.seq_num
     _getById('eventName').textContent = filename
     const oldImg = _getById('eventImage')
-    const newImg = document.createElement('img')
+    // create new repsonsive <img> element
```
Comment (sebastian-aranda) [inline src/js/pages/current.js:24]:
```suggestion
    // create new responsive <img> element
```

**Review by sebastian-aranda — CHANGES_REQUESTED** [review]
Just found a typo, the rest looks good to me :+1:.

File: src/js/pages/current.js (hunk 1 above)
Comment (ugyballoons) [reply src/js/pages/current.js:24]: Eagle eyes! Thanks

**Review by sebastian-aranda — APPROVED** [review]
Looks good to me :+1:.

**Pull request merged**